selftests/unit/junit-4.xsd

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
```

Comment:
What's the licensing terms for this file? I assume you've copied it from somewhere. I found multiple copies of it online, usually without any licensing information. Exception for this one, explicitly released under LGPL:  http://windyroad.com.au/dl/Open%20Source/JUnit.xsd (not sure if it's junit-v4)